plugins/og-image-generator/fonts/index.ts

```diff
@@ -0,0 +1,41 @@
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+import { type Font } from 'satori';
+
+type FontMetadata = Omit<Font, 'data'> & { filename: string };
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+/**
+ * Satori requires font metadata to be passed in order to render text with the correct font.
+ * This metadata includes the font name, filename, and weight.
+ * The `name` will be used in the `font-family` CSS property as font faces.
+ *
+ * Also, satori does not support variable fonts, so static fonts must be provided separately
+ * for each weight.
+ *
+ * Note: The `NotoSans` font is used for CJK characters.
+ */
+const fontsMetadata = [
+  { name: 'Inter', filename: 'Inter_18pt-Regular.ttf', weight: 400 },
```

Comment:
<img width="846" alt="image" src="https://github.com/user-attachments/assets/fd020879-1231-4753-b852-c40561f3b05c">

this is on their readme

Reply:
Ah, I see. You are actually suggesting using `fetch` to load Google fonts from URL.